### ISSUE TITLE
Container with CSP and offline support

### ISSUE THREAD

**Issue by GAS85** [AykutSarac/jsoncrack.com#226]
### Feature

Jsoncrack editor will call multiple external domains (cloudflare, gstatic, googletagmanager, sentry.io) to fetch css, js, etc additional data. This is makes it hard to deploy on servers with limited CSP settings, and require additional Security and Privacy User agreement.
![изображение](https://user-images.githubusercontent.com/6813635/193911210-8b210605-1ffd-4216-a1ba-e8e533b9a920.png)

It would be nice to have kind of "offline" editor Version, e.g. managed via variables give admin opportunity to disable external sites coiling and fetch static data only from the container.

### Alternative solutions or implementations

Such approach is implemented in [swagger editor](https://github.com/swagger-api/swagger-editor#running-the-image-from-dockerhub) v4, where admin can even disable parts of the editor simply via variables.
Basically ether before to build container source code should be cleaned up from the external resources and replaced with local links, or by container start.

### Other context

E.g. how it was done for other project. https://github.com/amnuts/opcache-gui/issues/84

**Comment by mpern** [comment]
Asking the naive question:
is there a way to bundle (webpack, rollup ...) the whole thing and don't access any third party resources over the network at all?

**Comment by GAS85** [comment]
This is a point of this ticket - zero access to 3rd party resources.
